index.js

```diff
@@ -0,0 +1,94 @@
+const MainBuilder = function (value) {
+  this.value = value;
+};
+
+MainBuilder.prototype.plus = function (...args) {
+  this.value += args.reduce((acc, num) => acc + num);
+  return this;
+};
+
+MainBuilder.prototype.minus = function (args) {
```

Comment:
From my point of view, I would move minus, multiply and divide methods to string class, because they seems to be expected to work with strings mostly